src/opcode-code-table.h

```diff
@@ -24,7 +24,12 @@
 extern "C" {
 #endif
 
-#define WABT_OPCODE_CODE_TABLE_SIZE 65536
+#define WABT_OPCODE_CODE_TABLE_SIZE 131072
+
+/*
+ * Number of bits required to store an opcode
+ */
+#define MAX_OPCODE_BITS 9
```

Comment:
Is this an intentional part of the relaxed simd proposal, that the opcode space is now larger and a byte?

Reply:
The vector instructions were always using variable-length integer opcodes, but these are the only instructions in this implementation with opcodes above 0xff. This necessitates a change in the structure of the internal lookup table (and also unfortunately doubles its size to 128 kB).

Reply:
Just to be clear, adding relaxed SIMD introduces opcode whose encoding is > 0xff for the first time?

Reply:
Yes, to be precise we're [adding](https://github.com/WebAssembly/wabt/pull/1994/files#diff-61d33dba45369870124687df0c499690e95ce7336a06f3e2df45226e25034d2fR505) prefixed opcodes with suffixes over 0xff.